app/Http/Controllers/BoulderController.php

```diff
@@ -99,11 +99,13 @@ public function update(Request $request, $id)
      * Remove the specified resource from storage.
      *
      * @param int $id
-     * @return \Illuminate\Http\Response
+     * @return \Illuminate\Http\JsonResponse
+     * @throws \Exception
      */
     public function destroy($id)
     {
-        //
+        $success = Auth::user()->boulders()->find($id)->delete();
```

Comment:
A bit more efficient in this case would be to do:
`Auth::user()->boulders()->destroy($id)`
Then you don't first retrieve the boulder `find($id)` before you delete it.

Reply:
ah ok that makes now also more sense with the name destroy of the call

Reply:
I dont think this works 
i get an error on this..

Reply:
That's odd. Not important enough to find out why/how. :P